airflow/www/views.py

```diff
@@ -471,8 +471,8 @@ def chart_data(self):
             df = hook.get_pandas_df(
                 wwwutils.limit_sql(sql, CHART_LIMIT, conn_type=db.conn_type))
             df = df.fillna(0)
-        except Exception as e:
-            payload['error'] += "SQL execution failed. Details: " + str(e)
+        except Exception:
+            payload['error'] += "SQL execution failed. Contact your System Administrator for more details"
```

Comment:
We probably want to log this error, otherwise there's no info for the sysadmin to get :)

```suggestion
        except Exception:
            self.log.exception("Chart SQL execution failed")
            payload['error'] += "SQL execution failed. Contact your System Administrator for more details"
```

Ditto for other change.

Reply:
(Assumming `self.log` exists in this context.

Reply:
🤦 Whoops -- yeah, fixed in b606b87